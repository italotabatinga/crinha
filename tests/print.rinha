let function = fn () => {};

print ("Crinha!")
print (2)
print (true)
print (false)
print (function)
print ((1,2))
print (((1, 2), 3 + 5))