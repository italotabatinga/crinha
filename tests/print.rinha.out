Crinha!
2
true
false
<#function>
(1, 2)
((1, 2), 8)
